mobile/ios/ios-emulator-services.ts

```diff
@@ -39,6 +39,19 @@ class IosEmulatorServices implements Mobile.IEmulatorPlatformServices {
 		}).future<void>()();
 	}
 
+	public postDarwinNotification(notification: string): IFuture<void> {
+		let iosSimPath = require.resolve("ios-sim-portable");
+		let nodeCommandName = process.argv[0];
+
+		let opts = [ "notify-post", notification ];
+
+		if (this.$options.device) {
+			opts.push("--device", this.$options.device);
+		}
+
+		return this.$childProcess.exec(`${nodeCommandName} ${iosSimPath} ${opts.join(' ')}`);
```

Comment:
We've discussed the code with @Fatme and it seems the option cannot have whitespace, so ignore my comment.